alibi/explainers/ale.py

```diff
@@ -106,8 +106,12 @@ def explain(self, X: np.ndarray, features: Optional[List[int]] = None, min_bin_p
 
         Returns
         -------
+        explanation
             An `Explanation` object containing the data and the metadata of the calculated ALE curves.
+            See usage `examples`_ for details.
 
+            .. _examples:
+                https://docs.seldon.io/projects/alibi/en/latest/methods/ALE.html
```

Comment:
The formatting here is off on the built docs and the link doesn't work. I'm not exactly sure what the issue is, there are similar instances in `KernelShap` where some links have the same issue but others work as expected.

Reply:
This is probably some really annoying corner case of formatting, maybe @ascillitoe has come across this...

Reply:
Inline link works fine 
```
`examples <https://docs.seldon.io/projects/alibi/en/latest/methods/ALE.html>`_
```
which could be used as a workaround.

Reply:
OK so all RST targets live in the same namespace which both causes warnings in the sphinx build and makes links not work. The solution is to either use inline links (as above) or the following anonymous target syntax:
```
        explanation
            An `Explanation` object containing the data and the metadata of the calculated ALE curves.
            See usage examples__ for details.
            
            __ https://docs.seldon.io/projects/alibi/en/latest/methods/ALE.html
```